meta-mender-core/classes/mender-helpers.bbclass

```diff
@@ -263,15 +263,16 @@ def mender_get_extra_parts_offset(d):
 
 # Take the content from the rootfs that is going into the boot partition, coming
 # from MENDER_BOOT_PART_MOUNT_LOCATION, and merge with the files from
-# IMAGE_BOOT_FILES, following the format from the official Yocto documentation.
+# MENDER_IMAGE_BOOT_FILES, following the format from the Yocto documentation
+# for IMAGE_BOOT_FILES.
 mender_merge_bootfs_and_image_boot_files() {
     W="${WORKDIR}/bootfs.${BB_CURRENTTASK}"
     rm -rf "$W"
 
     cp -al "${IMAGE_ROOTFS}/${MENDER_BOOT_PART_MOUNT_LOCATION}" "$W"
 
     # Put in variable to avoid expansion and ';' being parsed by shell.
-    image_boot_files="${IMAGE_BOOT_FILES}"
+    image_boot_files="${MENDER_IMAGE_BOOT_FILES}"
```

Comment:
The problem is that this definition makes building a .wic image to do custom install work for a target (in our case, setting up secure boot and LUKS) impossible, because the files needed to do that conflict with files installed by Mender. Alternative suggestions on how to deal with this would be appreciated!

Reply:
Is the wic image built using a different image recipe? If so, does it work to modify `IMAGE_BOOT_FILES` in each recipe? In theory it should, though I have not tried this.

Reply:
I have managed to revert this commit, so it will get squashed away in our `dunfell` and `master` branches. This was indeed required by our usage of `IMAGE_BOOT_FILES`, which we have since revised to avoid the problem.